scripts/create-new-package.bash

```diff
@@ -22,20 +22,36 @@ echo -e "${TERMINAL_COLOR_USER_NOTICE}Your path is `pwd`. Is this your workspace
 echo -e "${TERMINAL_COLOR_USER_CONFIRMATION}If so press <ENTER> otherwise <CTRL>+C and start the script again from your source folder.${TERMINAL_COLOR_NC}"
 read
 
-echo -n -e "${TERMINAL_COLOR_USER_INPUT_DECISION}What type of package you want to create? (0 - standard, 1 - metapackage, 2 - subpackage) [0]:${TERMINAL_COLOR_NC}"
-read META
-META=${META:=0}
+echo -n -e ""
+echo -e "${TERMINAL_COLOR_USER_INPUT_DECISION}What type of package you want to create? (1 - standard, 2 - metapackage, 3 - subpackage):${TERMINAL_COLOR_NC}"
+select META in standard metapackage subpackage;
+do
+  case "$META" in
+        standard)
+            META="1"
```

Comment:
can we keep "0", "1", "2"? It somehow more natural that "0" is default

Reply:
This should be actually default, so we only get to check "1" and "2" here.

Reply:
going to look if the default value in select can be changed

Reply:
Should the other select also be changed?

Reply:
since 1 is the default for select i don't change this.